packages/botonic-cli/jest.config.js

```diff
@@ -1,7 +1,10 @@
 module.exports = {
-  roots: ['<rootDir>/src'],
+  preset: 'ts-jest',
+  testEnvironment: 'node',
+  testTimeout: 1000000,
```

Comment:
I think 1000000 is too high for timeout. Should we use a lower timeout?

Reply:
do the entire pipeline (which include install dependencies, build the project with webpack and so on) takes lot of time,  I tried to use lower timeouts but they were not useful to success.